inst/include/LocalCop/gaussian.hpp

```diff
@@ -14,6 +14,68 @@
 
 namespace LocalCop {
 
+  // TODO: This code is a TMB example, copied here to test how it works. Need to replace it with pgaussian.
+  /*
+    This class evaluates the marginal density of x where
+    u     ~ Normal( mu, sd^2 )
+    x | u ~ Binom ( n , plogis(u) )
+  */
+  template<class Float>
+  struct GaussBinomial_t {
+    typedef Float Scalar; // Required by integrate
+    Float x, n;           // Data
+    Float mu, sd;         // Parameters
+    Float u;              // Integration Variable
+    // Evaluate joint density of (u, x)
+    Float operator() () {
+      Float ans = 0;
+      ans += dnorm(u, Float(0.), Float(1.), true);
+      Float p = invlogit(sd * u + mu);
+      p = squeeze(p);
+      ans += x * log(p) + (n - x) * log(1. - p);
+      ans = exp(ans);
+      // Avoid NaNs in the gradient:
+      if (ans == 0) ans = 0;
+      // Avoid NaNs in the tail of the distribution:
+      using atomic::tiny_ad::isfinite;
+      if (!isfinite(ans)) ans = 0;
+      return ans;
+    }
+    // Integrate latent variable (u) out
+    Float marginal() {
+      using gauss_kronrod::mvIntegrate;
```

Comment:
You used `gauss_kronrod::mvIntegrate()` here, despite the fact that it's a 1D integration.  I suggest you stick with `gauss_kronrod::Integrate()` for now.  Ideally we will be able to avoid 2D integration here altogether, at least until we have much better integrands for computing the bivariate Gaussian and Student-t CDFs.